### DISCUSSION TITLE
Play CDN for v4.0

### DISCUSSION THREAD

**Discussion by fostrwski** [tailwindlabs/tailwindcss#15479] [Help] [answered]
Hi! Is there a way to use play cdn with tailwind@4.0? 
If not, do you have a rough estimate on when the play cdn might support v4?

**Comment by gnat** [comment, marked as answer]
Answered in https://github.com/tailwindlabs/tailwindcss/discussions/15644

https://unpkg.com/@tailwindcss/browser@4.0.0-beta.9/dist/index.mjs
